classify_genes.R

```diff
@@ -139,7 +169,8 @@ classification$general_class = rep("Varied", dim(classification)[1]) ## initiate
 classification$general_class[which(classification$core == classification$total)] = "Core"  ##always core
 classification$general_class[which(classification$inter == classification$total)] = "Intermediate" ## always inter
 classification$general_class[which(classification$rare == classification$total)] = "Rare" ## always rare
-classification$general_class[which(classification$total == 0)] = "Absent in large lineages"
+classification$general_class[which(classification$singleton == 1)] = "Singleton" ## always singleton
```

Comment:
isn't this always the case because a singleton is defined as being observed once?